examples/default/src/screens/apm/HttpScreen.tsx

```diff
@@ -0,0 +1,250 @@
+import React from 'react';
+import { useState } from 'react';
+import { ToastAndroid, Platform, ActivityIndicator, Alert } from 'react-native';
+import { ListTile } from '../../components/ListTile';
+import { Screen } from '../../components/Screen';
+
+export const HttpScreen: React.FC = () => {
+  const [loadingGet, setLoading] = useState(false);
+  const [loadingPost, setLoading2] = useState(false);
+  const [loadingDelete, setLoading3] = useState(false);
+  const [loadingPut, setLoading4] = useState(false);
+  const [loadingPatch, setLoading5] = useState(false);
+  const [loadingDownloadImage, setLoading6] = useState(false);
```

Comment:
Let's rename these state setters to more descriptive names following the common React naming conventions.

```suggestion
  const [loadingGet, setLoadingGet] = useState(false);
  const [loadingPost, setLoadingPost] = useState(false);
  const [loadingDelete, setLoadingDelete] = useState(false);
  const [loadingPut, setLoadingPut] = useState(false);
  const [loadingPatch, setLoadingPatch] = useState(false);
  const [loadingDownloadImage, setLoadingDownloadImage] = useState(false);
```